### ISSUE TITLE
Feature Request: Include namespace when using CLAIM_NAME as snapshot name

### ISSUE THREAD

**Issue by kimsandb** [miracle2k/k8s-snapshots#36]
If you have a persistent volume with same name in different namespaces its not directly clear from what namespace the snapshot is.
For example if you have test & prod namespaces with similar setups.
It could be specified in the snapshot name or perhaps as a label?

**Comment by miracle2k** [comment]
Good idea. A label is also possible. As for changing how the name is generated, it's slightly backwards-incompatible, but I think acceptable to just change the behaviour of USE_ CLAIM_NAME
